flink-runtime-web/src/main/java/org/apache/flink/runtime/webmonitor/history/HistoryServerArchiveFetcher.java

```diff
@@ -231,18 +243,20 @@ public void run() {
 	 * <p>For the display in the HistoryServer WebFrontend we have to combine these overviews.
 	 */
 	private static void updateJobOverview(File webDir) {
-		File webOverviewDir = new File(webDir, "overviews");
-		try (JsonGenerator gen = jacksonFactory.createGenerator(HistoryServer.createOrGetFile(webDir, "joboverview"))) {
+		File webOverviewDir = new File(webDir, "/jobs/");
```

Comment:
the slashes are unnecessary, and the directory should be explicitly created here